packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/overview/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Third-Party Risk Provider Integration Overview
+---
+
+<ApiLifecycle access="beta" /> This is a beta feature guide.
+
+The Okta Risk Engine evaluates authentication attempts by reviewing the risk score of the sign-in based on context and historical data. Using Okta Risk APIs, third-party Risk Providers can integrate with the Okta Risk Engine using a standard Okta service application. The third-party Risk Provider can send risk signals, or events, which can be used when calculating the authentication risk based on the risk policy configured in the Okta org. The risk signals are additionally logged as part of the System Log.
+
+This guide provides an example third-party Risk Provider implementation with your Okta org.
+
+>**Note**: Third party risk signals are received from Non-Okta Applications. You are not required to receive or utilize third party risk signals within Okta Risk Engine, but if you configure Okta Risk Engine to utilize third party risk signals, then you are consenting to Okta receiving and sharing data with the Non-Okta Application as necessary to provide this functionality. You may only utilize these third party risk signals if you are a customer of both Okta and the Non-Okta Application. Okta cannot guarantee continued partnerships or functionality with any Non-Okta Applications.
+
+### Prerequisites
+To use this guide, you need the following:
+
+- An Okta developer org. [Create an org for free](/signup/).
+- To enable your Okta developer org with the following feature flags (FF): `RISK_SCORING` and `CONSUME_EXTERNAL_RISK_SIGNALS`. Contact your Okta customer support representative to enable these features.
```

Comment:
We do not name feature flags in our docs. I'm not aware of any exceptions.

Reply:
Removed with reference to contact support.